cosmic_ray/mutating.py

```diff
@@ -49,26 +49,30 @@ def activation_record(self):
         """
         return self._activation_record
 
-    def visit_mutation_site(self, node, op):
+    def visit_mutation_site(self, node, op, count):
```

Comment:
The parameter name `count` is a bit confusing since it collides with with `self._count`; we're using the same word to mean two different things here. I think it would be better to call it `num_mutations` or something more explicit.